docs/content/operator/full-node/troubleshooting.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Troubleshooting
+description: Troubleshooting common issues with your IOTA full node.
+teams:
+  - iotaledger/node
+---
+
+If you encounter errors or your full node stops working, run the commands in the following section to resolve the issue.
+
+## Issues during setup
+
+<Tabs groupId="setup-method">
+    <TabItem label="Systemd" value="systemd">
+      #### `Error: cannot find -lpq`
```

Comment:
```suggestion
      ### `Error: cannot find -lpq`
```

Btw do we need to highlight it? Looks weird imo 🤔 I would just make it a normal header